os/iphone-default-sdk.cmake

```diff
@@ -0,0 +1,66 @@
+# This script sets the following variables :
+# IOS_SDK_VERSION : will contain the version number of the default iOS SDK (example : 11.0)
+# IPHONEOS_SDK_ROOT : full path to the SDK
+# IPHONEOS_ROOT
+# XCODE_DEVELOPER_ROOT
+
+if(DEFINED POLLY_IPHONE_DEFAULT_SDK_CMAKE)
+return()
+else()
+set(POLLY_IPHONE_DEFAULT_SDK_CMAKE 1)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/../utilities/polly_status_debug.cmake")
```

Comment:
Can be just `include(polly_status_debug)`.

Reply:
`CMAKE_MODULE_PATH` modified by `polly_common` module:
* https://github.com/ruslo/polly/blob/2f1b497df080164f459a7733069267e7eafff03c/utilities/polly_common.cmake#L13-L15